Source/ACE/Command/Handlers/DebugCommands.cs

```diff
@@ -876,5 +876,33 @@ public static void BarberShop(Session session, params string[] parameters)
             chain.AddAction(session.Player, () => session.Network.EnqueueSend(new GameEventStartBarber(session)));
             chain.EnqueueChain();
         }
+        /// <summary>
+        /// Debug console command to test the GetSpellFormula function.
+        /// </summary>
+        [CommandHandler("spell", AccessLevel.Admin, CommandHandlerFlag.ConsoleInvoke, 0, "Tests spell formula calculation")]
```

Comment:
That's why it is in debug commands--so I can debug the function. I've left it in here so I can debug the issue I noted.